common/src/main/java/org/apache/gravitino/dto/authorization/RoleDTO.java

```diff
@@ -186,9 +192,6 @@ public S withAudit(AuditDTO audit) {
     public RoleDTO build() {
       Preconditions.checkArgument(StringUtils.isNotBlank(name), "name cannot be null or empty");
       Preconditions.checkArgument(audit != null, "audit cannot be null");
-      Preconditions.checkArgument(
-          securableObjects != null && securableObjects.length != 0,
-          "securable objects can't null or empty");
```

Comment:
Is it still removable, or should be checked?

Reply:
Yes, if one role has one securable object about catalog, and we remove the catalog. the securable objects could be empty.